src/index.html

```diff
@@ -85,7 +85,7 @@ <h5>About Us</h5>
             <div class="col-sm-3 ">
                 <h5>Share on Social Media</h5>
                 <ul class="nav">
-                    <li class="nav-item mb-2"><a href="#" class="nav-link p-0 text-muted"><i
+                    <li class="nav-item mb-2"><a href="#" class="linedin.com"><i
```

Comment:
I don't think this would have the result you think it would.
The ``class`` attribute is meant to apply CSS styling to a specific element. In this case the `li` tag: https://www.w3.org/Style/CSS/Overview.en.html

I think you meant to change the `href` attribute to point to LinkedIn? Is that correct?
In this case, it might have been:

`<li class="nav-item mb-2"><a href="linkedin.com" class="nav-link p-0 text-muted"></li>`

Reply:
Hi Dina, I think you can close this. I took a look and what it was referring to was needed to add accessibility to the Anchor tags. In this case, it wanted an `aria-label` tag added.  I have done this in Pull Request 55 Along with other changes: https://github.com/nathandh/mywebclass-simulation/pull/55